components/modal/RegisterModal/RegisterForm/index.js

```diff
@@ -38,7 +38,6 @@ export default function RegisterForm({ onCancel }) {
       const errorSource = getErrorSource(invalid);
 
       if (errorSource === "email") {
-        console.info(`settings error on email: ${invalid.message}`);
         setError("email", { type: "manual", message: invalid.message });
       } else {
         setError("form", {
```

Comment:
I think this error is causing some unexpected problems. When there's an error on the form, I can't resubmit this form without refreshing the page:
<img width="552" alt="Screen Shot 2022-04-05 at 12 54 00 PM" src="https://user-images.githubusercontent.com/57107444/161839174-46489941-0476-4933-97c9-dbe586f7b626.png">

Reply:
I think I've fixed all the issues around errors and additional form submissions.